versions.props

```diff
@@ -8,6 +8,7 @@ org.apache.orc:* = 1.8.2
 org.apache.parquet:* = 1.12.3
 org.apache.pig:pig = 0.14.0
 com.fasterxml.jackson.*:* = 2.14.1
+com.google.code.findbugs:jsr305 = 3.0.2
```

Comment:
@jackye1995 and @amogh-jahagirdar, this should be a banned dependency that is replaced by stephenc's reimplementation. This is a 1.2.0 release blocker.